modules/virt-features-for-storage-matrix.adoc

```diff
@@ -6,32 +6,89 @@
 = {VirtProductName} storage feature matrix
 
 .{VirtProductName} storage feature matrix
+ifdef::virt-features-for-storage[]
 [cols="40%,30%,30%",options="header"]
 |===
-||Virtual machine live migration  |Host-assisted virtual machine disk cloning
+|
+|Virtual machine live migration
+|Host-assisted virtual machine disk cloning
+endif::[]
+ifdef::virt-importing-rhv-vm,virt-importing-vmware-vm[]
+[cols="50%,50%",options="header"]
```

Comment:
If you add "caption=" to the options, it won't add the redundant "Table 1." to the caption. ie:
[cols="50%,50%",options="header",caption=]